tests/pkg_mbedtls/Makefile

```diff
@@ -0,0 +1,15 @@
+include ../Makefile.tests_common
+
+USEPKG += mbedtls
+USEMODULE += mbedtls_entropy
+
+# You can disable single entropy source to be used with the entropy module
+# DISABLE_MODULE += mbedtls_entropy_source_hwrng
+
+include $(RIOTBASE)/Makefile.include
+
+ifneq (,$(filter mbedtls_entropy_source_adc,$(USEMODULE)))
+	ifndef CONFIG_KCONFIG_USEMODULE_ENTROPY_SOURCE_ADC_NOISE
+		CFLAGS += -DCONFIG_ENTROPY_SOURCE_ADC_HMIN=65536
+	endif
+endif
```

Comment:
Why don't you put this in the package Makefile.include ?

Reply:
To avoid mis-configuration of this entropy source, we set an invalid default value. Hence, a user must evaluate and configure the module according to the hardware in use and the deployment. This line prevents failures during testing.